### PR TITLE
Adding utility function to test importing all submodules from packages

### PR DESCRIPTION
**Pull request by bsipocz**
This is the infrastructure for @eteq's feature request of adding import only travis tests https://github.com/astropy/astropy/issues/7939 based on @astrofrog's comment.

### CODE REVIEW

File: utils/import_submodules.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import sys
+import pkgutil
+import importlib
+
+
+def import_submodules(package, recursive=True):
+    """
+    Import all submodules of a module, recursively, including subpackages
+    Original form is from StackOverflow.
```
Comment (astrofrog) [inline utils/import_submodules.py]: Provide link

**Comment by astrofrog** [comment]
Currently this imports the tests modules, which I think we won't want in astropy, so I think we'll need to have a way to enable/disable this.

File: utils/import_submodules.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        https://stackoverflow.com/a/25562415
+
+    Parameters
+    ----------
+
+    package : str or module name
+        name of the package or the imported package to import the
+        submodules from.
+    skip_modules : coma separated str or None
+        Coma separated string of module names to be skipped.
```
Comment (astrofrog) [inline utils/import_submodules.py]: typo: comma

Comment (bsipocz) [reply utils/import_submodules.py]: I'm happy that I need to rebase it anyway ;)

**Pull request merged**